pyccel/ast/builtin_methods/dict_methods.py

```diff
@@ -60,6 +60,16 @@ def dict_obj(self):
         """
         return self._dict_obj
 
+    @property
+    def modified_args(self):
+        """
+        Return an iterable of all the arguments which may be modified by this function.
+
+        Return an iterable of all the arguments which may be modified by this function.
+        This is notably useful in order to determine the constness of arguments.
```

Comment:
A tuple is iterable, but why not calling it by name?
```suggestion
        Return a tuple of all the arguments which may be modified by this function.

        Return a tuple of all the arguments which may be modified by this function.
        This is notably useful in order to determine the constness of arguments.
```